### PR TITLE
[TACHYON-801] Create utility function to reset master client

### PR DESCRIPTION
**Pull request by fanshiqing**
https://tachyon.atlassian.net/browse/TACHYON-801

### CODE REVIEW

File: servers/src/main/java/tachyon/worker/block/BlockWorker.java [hunk 1]
```diff
@@ -163,9 +161,6 @@ public BlockWorker() throws IOException {
     // Setup user metadata mapping
     // TODO: Have a top level register that gets the worker id.
     long workerId = mBlockMasterSync.getWorkerId();
-    String tachyonHome = mTachyonConf.get(Constants.TACHYON_HOME);
-    String ufsAddress =
-        mTachyonConf.get(Constants.UNDERFS_ADDRESS);
```
Comment (calvinjia) [inline servers/src/main/java/tachyon/worker/block/BlockWorker.java]: Thanks for getting rid of these lines, but in the future, could you try to keep the changes isolated to addressing the ticket?

**Comment by calvinjia** [comment]
+1 LGTM

File: servers/src/main/java/tachyon/worker/block/BlockWorker.java (hunk 1 above)
Comment (fanshiqing) [inline servers/src/main/java/tachyon/worker/block/BlockWorker.java]: @calvinjia I see! Thanks for your advice! That's will really make sense for others to better review!

**Comment by uronce-cc** [comment]
+1 LGTM

**Pull request merged**